drivers/usbc/tcpc/ucpd_stm32.c

```diff
@@ -15,10 +15,15 @@ LOG_MODULE_REGISTER(ucpd_stm32, CONFIG_USBC_LOG_LEVEL);
 #include <soc.h>
 #include <stddef.h>
 #include <zephyr/math/ilog2.h>
-#include <stm32g0xx_ll_system.h>
+#include <stm32_ll_system.h>
+#include <zephyr/drivers/pinctrl.h>
 
 #include "ucpd_stm32_priv.h"
 
+PINCTRL_DT_INST_DEFINE(0);
+static const struct pinctrl_dev_config *usbc_pcfg =
+					PINCTRL_DT_INST_DEV_CONFIG_GET(0);
```

Comment:
This looks a little strange - @gmarull could you please sanity check this?

Reply:
driver seems to be multi-instance, however, this breaks multi-instance support. Please move this to `TCPC_DRIVER_INIT` and store a reference to `drv_config_##inst`

Reply:
Done